api/api/error_handler.py

```diff
@@ -244,4 +244,4 @@ async def content_size_handler(request: ConnexionRequest, exc: ContentSizeExceed
         "detail": str(exc)
     }
     return json_response(data=problem, pretty=request.query_params.get('pretty', 'false') == 'true',
-                         status_code=413, content_type=ERROR_CONTENT_TYPE)
+                         status_code=413)
```

Comment:
The same here, it should also be restored:

![image](https://github.com/wazuh/wazuh/assets/23361101/b127a3e1-589b-46b8-86d7-45c0f16b89a3)